### DISCUSSION TITLE
Supported formats running in a Linux Kubernetes cluster

### DISCUSSION THREAD

**Discussion by nick-42** [dlemstra/Magick.NET#693] [Help] [answered]
Hi. I'm using Magick.NET in an ASP.NET Core 3 app running in a Linux Kubernetes cluster in Azure.

It works really well for jpg/png/tiff files, but I'm getting stuck trying to process other file formats including svgs and raw camera images.

Is this supported ?

If it is supported, I could use a little help please :) Any documentation or pointers would be great!

Thanks

**Comment by dlemstra** [comment]
Raw images are supported and that should give you the same result as on Windows.

SVG is supported but you will end up using the MSVG coder and that is not a good reader. Currently there is no support for using the librsvg library but that might be available some time in the future.

**Comment by nick-42** [comment]
Thanks for your quick reply.

Yes, raw files are working now thanks - I wasn't setting the format explicitly on some of them.

For svgs, would it be a big job to support librsvg ? Happy to help if I can ?

**Reply by dlemstra** [reply]
The first steps have been made. I added support for pango yesterday and adding librsvg should be possible them. This needs to be done in the Magick.Native project. How familiar are you with building c/c++ libraries on Linux/macOS?

**Comment by dlemstra** [comment, marked as answer]
A new release has just been published and it includes support for librsvg on Linux and macOS.

**Reply by nick-42** [reply]
Works great! Thanks Dirk.

**Comment by fkucuk** [comment]
> Hi. I'm using Magick.NET in an ASP.NET Core 3 app running in a Linux Kubernetes cluster in Azure.
> 
> It works really well for jpg/png/tiff files, but I'm getting stuck trying to process other file formats including svgs and raw camera images.
> 
> Is this supported ?
> 
> If it is supported, I could use a little help please :) Any documentation or pointers would be great!
> 
> Thanks

@nick-42 

I am trying to use Magick.NET-Q8-x64 package on Azure (AKS). But it doesn't work. It returns some weird characters.

 Do you have any suggestions?
